module/VuFind/tests/integration-tests/src/VuFindTest/Mink/BasicSearchTest.php

```diff
@@ -58,4 +58,97 @@ public function testOutOfBoundsPage()
             trim($this->findCss($page, '.search-stats')->getText())
         );
     }
+
+    /**
+     * Test simple top pagination
+     *
+     * @return void
+     */
+    public function testSimpleTopPagination(): void
+    {
+        $session = $this->getMinkSession();
+        $session->visit($this->getVuFindUrl() . '/Search/Results');
+        $page = $session->getPage();
+
+        // No paginator unless configured:
+        $this->unFindCss($page, '.search-header .pagination');
+        $this->unFindCss($page, '.search-header .pagination-simple');
+
+        // Enable pagination:
+        $this->changeConfigs(
+            [
+                'searches' => [
+                    'General' => [
+                        'top_paginator' => 'simple',
+                    ],
+                ],
+            ]
+        );
+
+        $session->visit($this->getVuFindUrl() . '/Search/Results');
+
+        // No prev page on first page:
+        $this->unFindCss($page, '.search-header .pagination-simple .page-prev');
+
+        $secondPage = $this->findCss($page, '.search-header .pagination-simple .page-next');
+        $secondPage->click();
+        $this->waitForPageLoad($page);
+        // Prev page now present:
+        $this->findCss($page, '.search-header .pagination-simple .page-prev');
```

Comment:
Is it worth clicking on the prev button and asserting that it returns you to the previous page, so that both buttons are tested?

Reply:
Good idea, now added.